dspace-server-webapp/src/main/java/org/dspace/app/rest/utils/Utils.java

```diff
@@ -668,7 +669,10 @@ void embedRelFromRepository(HALResource<? extends RestAddressableModel> resource
                 Object linkedObject = method.invoke(linkRepository, null, contentId, null, projection);
                 resource.embedResource(rel, wrapForEmbedding(resource, linkedObject, link, oldLinks));
             } catch (InvocationTargetException e) {
-                if (e.getTargetException() instanceof RuntimeException) {
+                // Can't do this beforehand because we lack information to call Evaluators
+                if (e.getTargetException() instanceof AccessDeniedException) {
+                    log.warn("Tried fetching resource: " + linkRest.name() + " for DSpaceObject with ID: " + contentId);
```

Comment:
Should we return an error here too?  I'm not sure I understand why this is just a warning if you've received an `AccessDeniedException`.  The comment above this is unclear to me though, so I'm not sure what to recommend here.

Reply:
Raf has added details in https://github.com/DSpace/DSpace/commit/34bf7dd76b09bd20226e8b4a2317bae472eaf8d7#diff-6fdce65db891f81afe28b714fe8ddddaR672

Reply:
Thanks for the updated comment / description in the code. This now makes sense & can remain as-is.